src/lib/testrail.ts

```diff
@@ -67,4 +67,28 @@ export class TestRail {
       })
       .catch(error => console.error(error));
   }
+
+  public closeRun(results: TestRailResult[]) {
+    axios({
+      method: 'post',
+      url: `${this.base}/close_run/${this.runId}`,
+      headers: { 'Content-Type': 'application/json' },
+      auth: {
+        username: this.options.username,
+        password: this.options.password,
+      },
+      data: JSON.stringify({ results }),
```

Comment:
Looking at the testrail docs it seems like `close_run` doesn't accept any data: https://www.gurock.com/testrail/docs/api/reference/runs#close_run

From what I understood it behaves same as the `delete_run` endpoint, so you can see `deleteRun` function for inspiration

Reply:
You are right @mickosav I didn't check the API docs before. applied the changes